growthbook/growthbook
# Add Svelte SDK

### Features and Changes

This pull request adds support for the Svelte framework in the library. Developers can now leverage the Svelte SDK to build user interfaces with GrowthBook. The Svelte SDK provides a set of components and utilities that enable seamless integration between the library and Svelte applications.

The key features and changes in this pull request are:

- Integration of the Svelte SDK into the library codebase.
- Addition of Svelte-specific components and utilities for enhanced development experience.
- Documentation updates to provide guidance on using the Svelte SDK and integrating the library with Svelte applications.

This pull request closes #507.

### Dependencies

This pull request has the following dependencies:

- Svelte v3.0 or higher

### Testing

To test these changes, follow these steps:

1. Clone the repository and switch to the branch containing this pull request.
2. Follow the [Getting Started instructions](https://github.com/growthbook/growthbook/blob/main/CONTRIBUTING.md#getting-started) to set up the general repository.
3. Use the command `yarn test` within the sdk-svelte folder.
4. Execute the existing test suite for the library to ensure no regressions have occurred.

### Additional Notes

This is a large set of changes and will require ample testing. The README has been provided with some examples, but is not fully complete yet.
